apps/frontend/assets/src/omg-plasma/config.js

```diff
@@ -3,7 +3,8 @@ export default {
   web3ProviderUrl: 'http://localhost:8545',
   watcherUrl: 'http://localhost:7434',
   childchainUrl: 'http://localhost:9656',
-  plasmaContractAddress: '0xb0df2ea778e5ddc6c85d25fbed335e67211a15b5',
+  plasmaContractAddress: '0xc887b342280bd175bb66e1d9323df0d52a83611a',
```

Comment:
Shouldn't we get this from some config somehow?

Reply:
this should be replaced with the blockchain config that is setup on backend @T-Dnzt

Reply:
@T-Dnzt can i merge this and make new pr implementing O's blockchain settings?